src/XGBoostSharp.Tests/XGBClassifierTest.cs

```diff
@@ -128,6 +130,45 @@ public void XGBClassifierTest_UsingDMatrixDirectly()
         TestUtils.AssertAreEqual(expected, actual);
     }
 
+    [TestMethod]
+    public void XGBClassifierTest_PredictionContributions()
+    {
+        var dataTrain = TestUtils.DataTrain;
+        var labelsTrain = TestUtils.LabelsTrain;
+        var dataTest = TestUtils.DataTest;
+
+        using var sut = CreateSut();
+        using var dMatrixTrain = new DMatrix(dataTrain, labelsTrain);
+        sut.Fit(dMatrixTrain);
+
+        using var dMatrixTest = new DMatrix(dataTest);
+        var actualContribs = sut.Predict(dMatrixTest, predContribs: true);
```

Comment:
@madskonradsen We should add separate tests for the remaining "advanced prediction results".